src/config/browser-options.js

```diff
@@ -7,6 +7,7 @@ const defaults = require("./defaults");
 const optionsBuilder = require("./options-builder");
 const utils = require("./utils");
 const { WEBDRIVER_PROTOCOL, DEVTOOLS_PROTOCOL, SAVE_HISTORY_MODE } = require("../constants/config");
+const { MIN_CHROME_VERSION_SUPPORT_BIDI } = require("../constants/browser");
```

Comment:
`MIN_CHROME_VERSION_BIDI_SUPPORT`?

Reply:
Hm, I don't know. I'll leave my own version, I like it better.

Moreover we already have `MIN_CHROME_VERSION_SUPPORT_ISOLATION`